src/main.rs

```diff
@@ -90,12 +91,26 @@ pub enum Message {
 }
 
 fn main() -> iced::Result {
+	let icon_bytes = include_bytes!("../assets/images/icon.ico");
```

Comment:
As you can see, I tinkered with this and failed :-)

Although I was doing the same, we should maybe not crash on icon failure?
Unlikely to fail if built correctly, but still - I plan to gradually remove all the unwraps/expects I can over time to reduce cases where it can fail.